roles/openbsd/templates/etc/dhcpd.conf

```diff
@@ -1,7 +1,7 @@
 authoritative;
 
 subnet 192.168.42.0 netmask 255.255.255.0 {
-	range {{ dhcpd_range }};
+	range 192.168.42.128 192.168.42.254;
```

Comment:
Having removed {{ dhcpd_range }}, this no longer has variables, and should be a copy file, not a template.